### ISSUE TITLE
Add geoip match

### ISSUE THREAD

**Issue by shartge** [MaxKellermann/ferm#4]
It would be nice if ferm provided a geoip match.
The patch is as simple as this:
```
--- ferm-2.3/src/ferm   2016-03-30 14:16:02.000000000 +0200
+++ /usr/sbin/ferm      2016-07-26 20:32:13.000000000 +0200
@@ -256,6 +256,7 @@
 add_match_def 'esp', qw(espspi!);
 add_match_def 'eui64';
 add_match_def 'fuzzy', qw(lower-limit=s upper-limit=s);
+add_match_def 'geoip', qw(!src-cc=s !dst-cc=s);
 add_match_def 'hbh', qw(hbh-len! hbh-opts=c);
 add_match_def 'helper', qw(helper);
 add_match_def 'hl', qw(hl-eq! hl-lt=s hl-gt=s);
```

**Comment by MaxKellermann** [comment]
Add documentation, please. (And make it a pull request.)

**Comment by MaxKellermann** [comment]
Merged https://github.com/MaxKellermann/ferm/pull/5

**Comment by shartge** [comment]
On 27.07.2016 11:06, Max Kellermann wrote:

> Merged #5 <https://github.com/MaxKellermann/ferm/pull/5>

Hmm, the PR is still open, you just closed the Issue without really
merging the change.

**Comment by reissmann** [comment]
I do have a usecase for this. Will this feature be available some time?
